packages/eslint-config-base/rules/exports.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  rules: {
+    // Do not prefer default exports
+    'import/prefer-default-export': 'off',
+
+    // Disallow default exports
+    'import/no-default-export': 'error',
+  },
+  overrides: [
```

Comment:
These rules should probably live inside the `eslint-config-react` config since they are only relevant for React UI apps. We don't want this for backend apps or other Node libraries.

Reply:
I was thinking the same thing, but I wondered if it would be easier to reason by keeping them in `exports.js` about since they are all directly related to this proposal.

Reply:
Since they are for React / UI items, we should move those overrides to that other lib. We don't manage `jsx` files in this base config.

Reply:
That makes sense too – I'll move the files over to our React plugin.